src/data/src/Eryph.StateDb/Specifications/OperationSpecs.cs

```diff
@@ -57,14 +58,16 @@ public GetAll(Guid tenantId, Guid[] roles, AccessRight requiredAccess, string ex
 
         public sealed class GetById : Specification<OperationModel>, ISingleResultSpecification<OperationModel>
         {
-            public GetById(Guid id, Guid tenantId, Guid[] roles, AccessRight requiredAccess,  string expanded, DateTimeOffset requestLogTimestamp)
+            public GetById(Guid id, AuthContext authContext, IEnumerable<Guid> sufficientRoles, string expanded, DateTimeOffset requestLogTimestamp)
             {
                 Query.Where(x => x.Id == id &&
```

Comment:
yes - originally I thought it would be a good idea if you can see ops that "touched" your project, executed by a admin. Later changed my mind, as this would not be consistent from the security concept, but forget to update GetById.